<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-M9HHM5I" ignored="1" name="Default test(failed)" tests="2" failures="1" timestamp="2021-02-28T09:15:03 CST" time="0.055" errors="0">
  <testcase name="test2" time="0.008" classname="demo.TestNGRetryFailedTestsDemo"/>
  <testcase name="test3" time="0.004" classname="demo.TestNGRetryFailedTestsDemo">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.TestNGRetryFailedTestsDemo.test3(TestNGRetryFailedTestsDemo.java:24)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test1" time="0.0" classname="demo.TestNGRetryFailedTestsDemo">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
